server/chat-plugins/announcements.ts

```diff
@@ -4,20 +4,31 @@
  */
 import {Utils} from '../../lib/utils';
 
+const MINUTE = 60000;
+
+export interface AnnouncementData {
+	activityId?: 'announcement';
```

Comment:
```suggestion
	activityId: 'announcement';
```

Shouldn't this be required?

Reply:
I can make it required, though it means adding it when /poll create is used. But i suppose that's not a big deal.

Reply:
Oh, you deal with that by having them be different types, not by adding it when `/poll create` is used.

Reply:
`AnnouncementData extends AnnouncementOptions`

Reply:
Oh, okay.